cli/cmd/plugin_publish_test.go

```diff
@@ -8,10 +8,13 @@ import (
 	"net/http/httptest"
 	"os"
 	"strings"
+	gosync "sync"
```

Comment:
`sync` seems to be a symbol in this package, could be a variable. Let me check locally...

oh, it's because of:
`func sync(cmd *cobra.Command, args []string) error`

And the codebase agrees with me as the `cmd` package as multiple places where `gosync "sync"` line exists, apparently.